WITHOUT_ORACLE.md

```diff
@@ -46,14 +54,12 @@ If you don't want to use Oracle as source for your HOMS instance.
       password: <%= ENV['HOMS_DB_PASSWORD'] %>
     ```
 
-8. Run `docker-compose` with custom config:
+9. Run `docker-compose` with custom config:
 
     ```bash
     docker-compose -f docker-compose.dev.yml up -d
     ```
 
-9. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
-
-10. (Optional) If you want to use demo processes, navigate to [Camunda](http://localhost:8080/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
+10. Navigate to [Camunda Admin](http://localhost:8766/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
 
-11. Log in at [HydraOMS](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+11. (Optional) If you want to use demo processes, navigate to [Camunda](http://localhost:8766/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
```

Comment:
You can remove this line. Seed application in camunda docker image creates user with certain email automatically https://github.com/latera/camunda-ext/blob/master/seed/src/main/groovy/org/camunda/latera/bpm/configuration/SeedApplication.groovy#L38